spec/support/coronavirus_landing_page_steps.rb

```diff
@@ -51,6 +55,11 @@ def when_i_visit_the_coronavirus_landing_page
     visit CORONAVIRUS_PATH
   end
 
+  def when_i_visit_a_coronavirus_landing_page_with_timeline_nation_applicability
+    allow(Rails.env).to receive(:production?).and_return(true)
```

Comment:
I'm a bit confused by this too as I'd have thought this disables the feature?

Reply:
No, setting this to true means that it doesn't try to load the temporary content item from "config/data" and instead loads the fixture file instead. Otherwise we'd be testing the wrong content item. It doesn't really matter in this particular case as both content items ("config/data" and fixture file) have the same contents.

Reply:
Oh I see 🤯 

Would we be able to avoid most of the mocking of `Rails.env.production?` by changing the controller condition to be if `@selected_country && Rails.env.development?`? It's the type of method I don't feel very comfortable with mocking given it likely has surprising effects to any unrelated code that queries the environment.

Reply:
To clarify, there isn't anything that disables the feature. If the timeline in the content item contains "national_applicability" the radio buttons are displayed. That means that we won't need to "turn it on".